.changelog/config.toml

```diff
@@ -15,5 +15,6 @@ entry_indent = 2
     # to Markdown) and a path relative to the project folder (i.e. relative to
     # the parent of the `.changelog` folder).
     [components.all]
+    globalfee = { name = "GlobalFee", path = "x/globalfee" }
```

Comment:
```suggestion
```

It doesn't exist any more.

Reply:
I was required to generate the changelog, but shouldn't be used anymore.